public/locales/en/translations.json

```diff
@@ -115,12 +115,33 @@
       "header": "Delete device",
       "description": "This action will delete the devices permanently from the platform.",
       "simulatedNotSupported": "Deleting simulated devices is not currently supported, this action will only be taken on the physical devices.",
+      "summaryHeader": "Summary",
       "affected": "Affected devices",
       "cancel": "Cancel",
       "apply": "Delete",
+      "pending": "Deleting devices... ",
       "applySuccess": "devices deleted successfully!",
       "close": "Close"
     }
+    ,
+    "new": {
```

Comment:
I think these names need to be more explicit for readability purposes. Something like `flyoutNewDevice` or wrap them in a `flyouts` container.

Reply:
That would be a change for all of the flyout translations, not just "new".  So let's do that in another changeset.